images/clang_tidy/Dockerfile.erb

```
@@ -0,0 +1,16 @@
+FROM sider/devon_rex_base:2.22.0
+
+COPY --chown=<%= chown %> images/clang_tidy/packages.list ${RUNNER_USER_HOME}/
+
+USER root
+RUN curl -fsSL https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add - && \
+    add-apt-repository "deb http://apt.llvm.org/buster/ llvm-toolchain-buster-10 main" && \
+    apt-get update && \
+    xargs apt-get install -qq -y --no-install-recommends < ${RUNNER_USER_HOME}/packages.list && \
+    apt-get clean && \
```

[question] Does removing `/var/lib/apt/lists/*` make sense?

```suggestion
    apt-get clean && \
    rm -rf /var/lib/apt/lists/* && \
```

E.g. https://github.com/sider/runners/blob/840af184a3e5b95e82661a8f5748a24e630ca5e1/images/cppcheck/Dockerfile.erb#L16

---

If we delete files in `/var/lib/apt/lists/`, we will fail to execute `apt-get install` at `deploy_packages` in `clang-tidy.rb`.
Otherwise, we must execute `apt-get update` before `apt-get install` at `deploy_packages`.

---

Make sense. 👍